### PR TITLE
Add fromHex helper

### LINKED ISSUE #56: Would it be possible to have some toAscii helper for bytes arguments?
Because `bytesX` arguments fit better than strings in some cases, I think it would be really helpful for the user to have a radspec helper that outputs the friendly ASCII or UTF8 value (as web3 utils has).

Example:
```sol
/**
 * @notice Add entity `@toAscii(_name)` with address `_addr` to the registry
 */
function addEntry(address _address, bytes32 _name) {...}
```

Those helpers would be pure functions, so they should not decrease overall safety

I can submit a PR for that if you find it is a good idea.

### PR DESCRIPTION
**Pull request by sembrestels**
Fixes #56.

### CODE REVIEW

File: src/helpers/fromHex.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { toUtf8, toDecimal } from 'web3-utils'
+export const fromHexToUtf8 = () =>
+  /**
+   * Returns the UTF-8 string representation of a given hex value
+   *
+   * @param {string} hex The hex string
+   * @return {radspec/evaluator/TypedValue}
+   */
+  hex => {
```
Comment (izqui) [inline src/helpers/fromHex.js]: I'd recommend keeping this as just one helper, `(hex, toDecimal = false) => `, which can then be used as `@fromHex(bytes, true)` for decimal or `@fromHex(bytes)` for UTF8. Alternatively we could pass the mode (utf8 or dec) as a string

Comment (sohkai) [reply src/helpers/fromHex.js]: Think it would make most sense to pass a string, with 'ascii', 'number', and 'utf8' as available options.

Comment (sembrestels) [reply src/helpers/fromHex.js]: Done.

**Comment by sembrestels** [comment]
Rebased on top of v1.2.0.

File: src/helpers/fromHex.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+export default () =>
+  /**
+   * Returns the string representation of a given hex value
+   *
+   * @param {string} hex The hex string
+   * @param {string} [to='utf8'] The type to convert the hex from (supported types: 'utf8', 'number', 'decimal')
+   * @return {radspec/evaluator/TypedValue}
+   */
+  (hex, to = 'utf8') => {
+    return {
```
Comment (izqui) [inline src/helpers/fromHex.js]: Little style thing: you can get rid of this return by using the () => ({ key: value }) syntax

Comment (sembrestels) [reply src/helpers/fromHex.js]: Done (also for other functions with the same issue).

**Pull request merged**